test/tests/fuel-cycle/fuel_cycle.i

```diff
@@ -0,0 +1,450 @@
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1000 # Abdou 2021 paper chosen for analysis
+  []
+  [residence6] #CPS
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 864000 #Abdou 1,4
+    #value 864000 (analysis case)
+  []
+  [residence7] #Vac
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1800 # 30 minutes (20-30 minutes)
+  []
+  [residence8] #FCU
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value =  4680 # 4 hours as per "reasonable approx" Abdou
+    #value = 1.3 h Day
+    #value = 5 h Coleman
+    #value = 8640 Abdou
+    #value = 86400 Abdou
+  []
+  [residence9] #ISS
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 7920
+  []
+  [tdecay]
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1.7828336471961835e-9
+  []
+  [tritium_burn_fraction]
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 0.0036
+  []
+  [tritium_burn_rate]
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 5.3125e-6 # 0.459 kg/day
+  []
+  [tritium_fueling_efficiency]
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 0.25
+  []
+  # These postprocessors exist to sum up the tritium inventory
+  #  across the entirety of the system
+  [tbz]
+    type = ScalarVariable
+    variable = T_01_BZ
+    execute_on = TIMESTEP_END
+  []
+  [T_TES]
+    type = ScalarVariable
+    variable = T_02_TES
+    execute_on = TIMESTEP_END
+  []
+  [T_FW]
+    type = ScalarVariable
+    variable = T_03_FW
+    execute_on = TIMESTEP_END
+  []
+  [T_DIV]
+    type = ScalarVariable
+    variable = T_04_DIV
+    execute_on = TIMESTEP_END
+  []
+  [T_HX]
+    type = ScalarVariable
+    variable = T_05_HX
+    execute_on = TIMESTEP_END
+  []
+  [T_CPS]
+    type = ScalarVariable
+    variable = T_06_CPS
+    execute_on = TIMESTEP_END
+  []
+  [T_VAC]
+    type = ScalarVariable
+    variable = T_07_vacuum
+    execute_on = TIMESTEP_END
+  []
+  [T_FCU]
+    type = ScalarVariable
+    variable = T_08_FCU
+    execute_on = TIMESTEP_END
+  []
+  [T_ISS]
+    type = ScalarVariable
+    variable = T_09_ISS
+    execute_on = TIMESTEP_END
+  []
+  [texo]
+    type = ScalarVariable
+    variable = T_10_exhaust
+    execute_on = TIMESTEP_END
+  []
+  [tsto]
+    type = ScalarVariable
+    variable = T_11_storage
+    execute_on = TIMESTEP_END
+  []
+  [total_tritium]
+    type = SumPostprocessor
+    values = 'tbz T_TES T_FW T_DIV T_HX T_CPS T_VAC T_FCU T_ISS texo tsto'
+  []
+[]
+#[Functions]
+#  [catch_five_year]
+#    type = PiecewiseLinear
+#    x = '0 157678999 157681001 864000000.0'
+#    y = '0 0 1 1'
+#  []
+#[]
```

Comment:
```suggestion
```
If that is not needed, let's remove it.

Reply:
I initially removed this, but I think it might be better to leave in. I added comments as to what it is for and why it is sometimes necessary.

Reply:
That makes sense. Let's keep it!